src/Outbox/StoreOutboxConsumer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Outbox;
+
+use Patchlevel\EventSourcing\EventBus\EventBus;
+use Patchlevel\EventSourcing\Store\OutboxStore;
+
+class StoreOutboxConsumer implements OutboxConsumer
```

Comment:
I think swapping the naming leads to better readability `OutboxConsumerStore`. But i feel like both options are somewhat misleading.

Reply:
I think it's a naming convention when you have an interface, like "OutboxConsumer" here, that you just prefix the implementations, so the suffix stays the same. I don't like the name either, but I don't know if your version is better :D